CHANGELOG.md

```diff
@@ -1,13 +1,58 @@
+## Fleet 4.55.0 (Aug 8, 2024)
+
+**NOTE:** Beginning with v4.55.0, Fleet no longer supports MySQL 5.7 because it has reached [end of life](https://mattermost.com/blog/mysql-5-7-reached-eol-upgrade-to-mysql-8-x-today/#:~:text=In%20October%202023%2C%20MySQL%205.7,to%20upgrade%20to%20MySQL%208.). The minimum version supported is MySQL 8.0.
```

Comment:
`8.0.36` right @dantecatalfamo @roperzh?

Reply:
Thanks for noticing; fixed!